packages/main/src/MessageStrip.ts

```diff
@@ -151,10 +69,86 @@ const metadata = {
  * @public
  * @since 0.9.0
  */
+@customElement("ui5-message-strip")
+@languageAware
+/**
+ * Fired when the close button is pressed either with a
+ * click/tap or by using the Enter or Space key.
+ *
+ * @event
```

Comment:
you need to add `@event sap.ui.webc.main.MessageStrip#suggestion-scroll`
for the public events, otherwise they won't make it to the api.json

this is one of the new things to keep in mind